test/integration/tests/staking.ts

```diff
@@ -201,6 +201,8 @@ describe('e2e-staking', function () {
       await tribalChief.connect(daoSigner).set(16, 500, ethers.constants.AddressZero, false);
       await tribalChief.connect(daoSigner).set(17, 250, ethers.constants.AddressZero, false);
 
+      // TOOD: Grant various autoRewardDistributors the relevant role
```

Comment:
should this test not just be deleted?

Reply:
Eventually I'd agree. I will probably delete it once all of the incentives is finished up - i.e. we pull the remaining TRIBE off the TribalChief and Votium contract. Until then, wanted to initialise these tests with expected state so they pass